### ISSUE TITLE
yolox_nano和tiny运行AP=0，其余正常，请问nano如何训练？

### ISSUE THREAD

**Issue by Baiyuetribe** [Megvii-BaseDetection/YOLOX#1355]
```bash
python -m yolox.tools.train -n yolox-s -d 1 -b 8 --fp16 -o --cache # 一切正常
```
当运行nano或tiny时ap=0.是不是还需要修改很多东西？
```
python -m yolox.tools.train -n yolox-nano -d 1 -b 8 --fp16 -o --cache
```

**Comment by FateScript** [comment]
For tiny, try batchsize = 16 for single gpu.

**Comment by FateScript** [comment]
Did you use cache during training? What if you use checkpoint we provided?

**Comment by MengtianLee** [comment]
I solved this issue by decreasing the initial learning rate.
